packages/create-liveblocks-app/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "create-next-liveblocks-app",
+  "version": "0.0.1",
+  "description": "Create a Liveblocks starter kit for Next.js",
+  "bin": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch",
+    "demo": "node scripts/demo.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/liveblocks/liveblocks.git#main"
+  },
+  "keywords": [
+    "next.js",
+    "liveblocks",
+    "starter",
+    "template",
+    "collaboration",
+    "real-time",
+    "multiplayer"
+  ],
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/liveblocks/liveblocks/issues"
+  },
+  "homepage": "https://github.com/liveblocks/liveblocks/tree/main#readme",
+  "devDependencies": {
+    "@types/command-line-args": "^5.2.0",
+    "@types/degit": "^2.8.3",
+    "@types/node": "^17.0.39",
+    "tsc": "^2.0.4",
```

Comment:
You won't need the `tsc` node package (it's [deprecated](https://www.npmjs.com/package/tsc)). When you depend on `typescript` , that'll be enough. It ships with the `tsc` executable built-in.